proto/zitadel/user/v2beta/user_service.proto

```diff
@@ -655,7 +655,9 @@ message AddHumanUserRequest{
       example: "\"minnie-mouse\"";
     }
   ];
+  // deprecated: use organization (if both are set, organization will take precedence)
   zitadel.object.v2beta.Organisation organisation = 3;
+  zitadel.object.v2beta.Organization organization = 11;
```

Comment:
The deprecation note is not shown in the [API documentation](https://docs-g0jlf8yty-zitadel.vercel.app/docs/apis/resources/user_service/user-service-add-human-user):

It does for the session response object somehow.

![image](https://github.com/zitadel/zitadel/assets/5411563/ac061390-3475-46a6-bc8f-d8b16ddab428)